rts/Rendering/Fonts/CFontTexture.cpp

```diff
@@ -595,11 +666,33 @@ CFontTexture::CFontTexture(const std::string& fontfile, int size, int _outlinesi
 	if (lineHeight <= 0)
 		lineHeight = 1.25 * (face->bbox.yMax - face->bbox.yMin);
 
+	// TODO: untested
+	if (canScale) {
+		fontDescender = fontDescender*(pixScale/normScale);
+		//fontDescender = fontDescender*pixScale ??
+		lineHeight = lineHeight*pixScale;
+	}
+
 	// has to be done before first GetGlyph() call!
-	CreateTexture(32, 32);
+	CreateTexture(32, 32, true);
 
 	// precache ASCII glyphs & kernings (save them in kerningPrecached array for better lvl2 cpu cache hitrate)
+	PreloadGlyphs();
+
+#endif
+}
 
+/***
+ *
+ * Preloads standard alphabet glyphs for a font
+ */
+void CFontTexture::PreloadGlyphs()
+{
+#ifndef HEADLESS
+	FT_Face face = *shFace;
+	// don't preload for non alphanumeric
+	if (!FT_Get_Char_Index(face, 65))
```

Comment:
The comment sounds a bit like we didn't want to preload alphanumeric glyphs, which reverses the meaning (as far as i understand). Also could use an alphanumeric char explicitly:
```suggestion
	// if given face doesn't contain alphanumerics, don't preload it
	if (!FT_Get_Char_Index(face, 'a'))
```

Reply:
you're right, wanted to say dont preload for non-alphanumeric fonts, but it can be misintrepreted

also, sorry this sneaked in... didn't really mean to add this part inside this PR, I think I'll move that to a separate PR ... I think that'd be great to have for next release while color fonts don't think will make it.

in any case I'll follow your suggestion